const/styles/pages/index.ts

```diff
@@ -31,7 +31,7 @@ export const Section = styled.section<{ hero?: boolean, breakMedium?: boolean, c
   ${({ hero, breakMedium }) => (hero || breakMedium) && `
     margin: 0 auto;
     min-height: calc(100vh - 8.1rem);
-    padding-top: 3.6rem;
+    padding-top: 150px;
```

Comment:
Please use `rem` values. 

1 rem = 16px (default on body/html, using the font-size).

So 150px = 15rem